src/gui/folder.cpp

```diff
@@ -46,8 +46,10 @@
 #include <QPushButton>
 #include <QApplication>
 
+#ifndef VERSION_C
+#define VERSION_C
 static const char versionC[] = "version";
-
+#endif
```

Comment:
please can you replace them by the following code
```
namespace {
constexpr auto versionC= "version";
}
```
can you also apply the same solution to the few other occurrences

Reply:
please ignore that comment
it is indeed not compiling
so please use
```
namespace {
#ifndef VERSION_C
#define VERSION_C
constexpr auto versionC= "version";
#endif
}
```